crates/holochain_zome_types/src/zome/inline_zome.rs

```diff
@@ -28,7 +28,7 @@ pub type InlineCoordinatorZome = InlineZome<CoordinatorZomeMarker>;
 /// An InlineZome, which consists
 pub struct InlineZome<T> {
     /// Inline zome type marker.
-    _t: PhantomData<T>,
+    pub _t: PhantomData<T>,
```

Comment:
Hmm maybe remove this one as this is never actually used in the wasm. So it can remain private